templates/accessibility.html

```diff
@@ -21,7 +21,7 @@ <h1>Accessibility guidelines</h1>
 <div class="p-strip">
   <div class="row">
     <div class="col-6">
-      <h3 class="p-heading--2">Ensuring conformance</h3>
+      <h2 class="p-heading--2">Ensuring conformance</h2>
```

Comment:
I guess we don't need the class if we switch to h2?

Reply:
:facepalm: